packages/types/surveys.ts

```diff
@@ -487,8 +470,7 @@ export const ZSurvey = z.object({
   status: ZSurveyStatus,
   displayOption: ZSurveyDisplayOption,
   autoClose: z.number().nullable(),
-  triggers: z.array(z.string()),
-  inlineTriggers: ZSurveyInlineTriggers.nullable(),
+  triggers: z.array(ZActionClass),
```

Comment:
this might feel like a nitpick, but having triggers as `z.array(z.object({actionClass: ZActionClass}))` makes more sense as it is coherent with the database schema 😓